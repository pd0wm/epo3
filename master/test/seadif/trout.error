This is the file 'seadif/trout.error' which was created by
the program 'trout' on Thu Dec 12 10:53:18 2013
Trout was verifying the netlist description of
your circuit circuit 'controller' with the current layout in seadali.
The following nets in the layout are not according to
the netlist description of circuit 'controller'. They are also
marked in the layout.
------------------------- list of errors -------------------------
@@@@@ entirely unconnected net:  'n742' (2 terminals)
@@@@@ entirely unconnected net:  'n673' (2 terminals)
@@@@@ partially unconnected net: 'n533' (1 unconnects among 3)
@@@@@ entirely unconnected net:  'n618' (2 terminals)
@@@@@ partially unconnected net: 'n462' (1 unconnects among 2)
@@@@@ partially unconnected net: 'lut_y_3_0_0' (1 unconnects among 2)
@@@@@ partially unconnected net: 'n433' (1 unconnects among 4)
@@@@@ partially unconnected net: 'n678' (1 unconnects among 2)
@@@@@ partially unconnected net: 'n542' (3 unconnects among 6)
@@@@@ partially unconnected net: 'n690' (1 unconnects among 2)
@@@@@ entirely unconnected net:  'n399' (2 terminals)
@@@@@ entirely unconnected net:  'n520' (2 terminals)
@@@@@ entirely unconnected net:  'n529' (2 terminals)
@@@@@ entirely unconnected net:  'n575' (2 terminals)
@@@@@ partially unconnected net: 'n565' (1 unconnects among 2)
@@@@@ partially unconnected net: 'n686' (1 unconnects among 4)
@@@@@ partially unconnected net: 'n694' (1 unconnects among 2)
@@@@@ partially unconnected net: 'inv_inputs_5_port' (3 unconnects among 5)
@@@@@ entirely unconnected net:  'n455' (4 terminals)
@@@@@ partially unconnected net: 'n475' (7 unconnects among 9)
@@@@@ entirely unconnected net:  'n525' (2 terminals)
@@@@@ partially unconnected net: 'n663' (1 unconnects among 2)
@@@@@ entirely unconnected net:  'n593' (2 terminals)
@@@@@ partially unconnected net: 'n671' (1 unconnects among 6)
@@@@@ partially unconnected net: 'n708' (31 unconnects among 34)
@@@@@ entirely unconnected net:  'n457' (4 terminals)
@@@@@ entirely unconnected net:  'n483' (2 terminals)
@@@@@ partially unconnected net: 'n484' (1 unconnects among 3)
@@@@@ partially unconnected net: 'n622' (4 unconnects among 8)
@@@@@ partially unconnected net: 'n656' (1 unconnects among 3)
@@@@@ entirely unconnected net:  'n713' (2 terminals)
@@@@@ partially unconnected net: 'n486' (6 unconnects among 8)
@@@@@ partially unconnected net: 'n624' (5 unconnects among 6)
@@@@@ entirely unconnected net:  'n672' (2 terminals)
@@@@@ entirely unconnected net:  'n472' (7 terminals)
@@@@@ entirely unconnected net:  'n606' (2 terminals)
@@@@@ partially unconnected net: 'n401' (8 unconnects among 9)
@@@@@ entirely unconnected net:  'n452' (7 terminals)
@@@@@ entirely unconnected net:  'n662' (2 terminals)
@@@@@ partially unconnected net: 'n582' (1 unconnects among 2)
@@@@@ partially unconnected net: 'n710' (1 unconnects among 3)
@@@@@ entirely unconnected net:  'n411' (2 terminals)
@@@@@ entirely unconnected net:  'n424' (7 terminals)
@@@@@ entirely unconnected net:  'n506' (4 terminals)
@@@@@ entirely unconnected net:  'n527' (4 terminals)
@@@@@ partially unconnected net: 'n440' (5 unconnects among 7)
@@@@@ entirely unconnected net:  'n714' (2 terminals)
@@@@@ entirely unconnected net:  'n528' (7 terminals)
@@@@@ entirely unconnected net:  'n477' (16 terminals)
@@@@@ entirely unconnected net:  'n454' (4 terminals)
@@@@@ entirely unconnected net:  'n602' (2 terminals)
@@@@@ entirely unconnected net:  'n551' (2 terminals)
@@@@@ entirely unconnected net:  'n490' (2 terminals)
@@@@@ partially unconnected net: 'n698' (1 unconnects among 2)
@@@@@ entirely unconnected net:  'next_state_4_port' (2 terminals)
@@@@@ partially unconnected net: 'check_empty' (14 unconnects among 15)
@@@@@ partially unconnected net: 'cur_state_1_port' (7 unconnects among 11)
@@@@@ entirely unconnected net:  'n421' (4 terminals)
@@@@@ entirely unconnected net:  'n461' (2 terminals)
@@@@@ entirely unconnected net:  'n442' (4 terminals)
@@@@@ entirely unconnected net:  'n628' (2 terminals)
@@@@@ partially unconnected net: 'n654' (1 unconnects among 2)
@@@@@ entirely unconnected net:  'n683' (2 terminals)
@@@@@ entirely unconnected net:  'n561' (4 terminals)
@@@@@ entirely unconnected net:  'n513' (11 terminals)
@@@@@ entirely unconnected net:  'n570' (4 terminals)
@@@@@ entirely unconnected net:  'n572' (2 terminals)
@@@@@ entirely unconnected net:  'n419' (2 terminals)
@@@@@ partially unconnected net: 'n437' (14 unconnects among 16)
@@@@@ entirely unconnected net:  'n515' (4 terminals)
@@@@@ entirely unconnected net:  'n706' (16 terminals)
@@@@@ partially unconnected net: 'lut_piece_type_2_0_1' (4 unconnects among 5)
@@@@@ entirely unconnected net:  'n733' (4 terminals)
@@@@@ entirely unconnected net:  'n596' (2 terminals)
@@@@@ partially unconnected net: 'cur_timer_1_time' (10 unconnects among 11)
@@@@@ entirely unconnected net:  'n502' (11 terminals)
@@@@@ entirely unconnected net:  'n429' (2 terminals)
@@@@@ partially unconnected net: 'n609' (14 unconnects among 15)
@@@@@ partially unconnected net: 'cur_future_piece_1_port' (1 unconnects among 2)
@@@@@ partially unconnected net: 'n474' (31 unconnects among 34)
@@@@@ entirely unconnected net:  'n557' (4 terminals)
@@@@@ partially unconnected net: 'lut_piece_type_2_0_2' (5 unconnects among 6)
@@@@@ entirely unconnected net:  'n625' (2 terminals)
@@@@@ entirely unconnected net:  'n647' (7 terminals)
@@@@@ entirely unconnected net:  'n703' (2 terminals)
@@@@@ entirely unconnected net:  'n550' (7 terminals)
@@@@@ entirely unconnected net:  'n615' (4 terminals)
@@@@@ entirely unconnected net:  'n426' (4 terminals)
@@@@@ entirely unconnected net:  'n709' (11 terminals)
@@@@@ entirely unconnected net:  'n712' (7 terminals)
@@@@@ partially unconnected net: 'draw_score_draw' (4 unconnects among 5)
@@@@@ entirely unconnected net:  'n511' (2 terminals)
@@@@@ entirely unconnected net:  'n668' (4 terminals)
@@@@@ entirely unconnected net:  'n584' (7 terminals)
@@@@@ entirely unconnected net:  'n707' (37 terminals)
@@@@@ entirely unconnected net:  'n638' (16 terminals)
@@@@@ entirely unconnected net:  'n682' (2 terminals)
@@@@@ partially unconnected net: 'n470' (1 unconnects among 2)
@@@@@ partially unconnected net: 'cur_state_0_port' (17 unconnects among 18)
@@@@@ entirely unconnected net:  'n655' (4 terminals)
@@@@@ partially unconnected net: 'cur_future_piece_2_port' (1 unconnects among 2)
@@@@@ entirely unconnected net:  'n597' (11 terminals)
@@@@@ partially unconnected net: 'n697' (36 unconnects among 37)
@@@@@ partially unconnected net: 'n702' (2 unconnects among 4)
@@@@@ entirely unconnected net:  'n432' (2 terminals)
@@@@@ entirely unconnected net:  'n614' (4 terminals)
@@@@@ entirely unconnected net:  'n558' (7 terminals)
@@@@@ entirely unconnected net:  'n376' (2 terminals)
@@@@@ entirely unconnected net:  'n414' (4 terminals)
@@@@@ partially unconnected net: 'n451' (18 unconnects among 22)
@@@@@ entirely unconnected net:  'n587' (7 terminals)
@@@@@ entirely unconnected net:  'n616' (2 terminals)
@@@@@ entirely unconnected net:  'n634' (4 terminals)
@@@@@ partially unconnected net: 'n632' (3 unconnects among 4)
@@@@@ partially unconnected net: 'draw_erase_ready' (7 unconnects among 10)
@@@@@ entirely unconnected net:  'n684' (2 terminals)
@@@@@ entirely unconnected net:  'n744' (2 terminals)
@@@@@ entirely unconnected net:  'n717' (2 terminals)
@@@@@ partially unconnected net: 'n571' (7 unconnects among 8)
@@@@@ entirely unconnected net:  'n566' (11 terminals)
@@@@@ entirely unconnected net:  'n598' (16 terminals)
@@@@@ partially unconnected net: 'n588' (14 unconnects among 15)
@@@@@ entirely unconnected net:  'inv_inputs_0_port' (4 terminals)
@@@@@ partially unconnected net: 'rst' (66 unconnects among 70)
@@@@@ entirely unconnected net:  'next_state_2_port' (2 terminals)

WARNING: 125 out of the 447 nets are not properly connected.
         In the layout each of the unconnects is marked by
         an arrow followed by the name of the net: <=-<net_name>
------------------------------------------------------------------
lock
