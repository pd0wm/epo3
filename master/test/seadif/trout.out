>>>>>> trout: automatic router for Sea-of-Gates           <<<<<<
>>>>>> (c) 1995 Patrick Groeneveld, Delft Univ. of Techn. <<<<<<
------ reading image description file '/data/public/common/software/nelsis/5.4.1.1/share/lib/celllibs/fishbone/image.seadif' ------
------ reading placement 'Tmp_Cell_(top_level2(top_level2(test)))' ------
------ building data structure  ------
WARNING: No circuit for layout instance 'osc10_1'
         (cell 'osc10'): it will not be routed
------ making grid  ------
------ Busy highlighting net ------
------ Writing 'Tmp_Cell_(top_level2(top_level2(test)))' ------
------ Trout: task completed on Fri Dec 13 23:21:20 2013
 terminate
